platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java

```diff
@@ -166,9 +165,9 @@ public class Browser {
 
     private static final String STARTUP_MESSAGE =
             """
-              //////////////////////
-             // Node is Starting //
-            //////////////////////""";
+                      //////////////////////
```

Comment:
This extra indent changes the string that's output.  Might want to check format settings to leave text blocks alone.

Reply:
Did not find the setting, had to added `@formatter:off` to maintain the change